pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/HLRealtimeSegmentDataManager.java

```diff
@@ -258,7 +258,7 @@ public void run() {
           // lets convert the segment now
           RealtimeSegmentConverter converter =
               new RealtimeSegmentConverter(realtimeSegment, tempSegmentFolder.getAbsolutePath(), schema,
-                  realtimeSegmentZKMetadata.getTableName(), timeColumnName, realtimeSegmentZKMetadata.getSegmentName(),
+                  tableName, timeColumnName, realtimeSegmentZKMetadata.getSegmentName(),
```

Comment:
tablenameWithType?

Reply:
Both `LLRealtimeSegmentDataManager` and `HLRealtimeSegmentDataManager` historically uses `tableName`. I can update this to `tableNameWithType` or `rawTableName` after checking.

Reply:
updated.